extensions/grpc/protoc/src/main/resources/MutinyBean.mustache

```diff
@@ -0,0 +1,63 @@
+{{#packageName}}
```

Comment:
We should probably write something in the log (if not done already) indicating what has been generated.

Reply:
Well, we do mention the generated bean here: https://github.com/quarkusio/quarkus/pull/17342/files#diff-3ee859718459e7dbace610702235f89b4dfc85eb25ce6f4a6ed24ec729ffdc2cR113 but in the guide we only mention the important parts: https://github.com/quarkusio/quarkus/pull/17342/files#diff-3ee859718459e7dbace610702235f89b4dfc85eb25ce6f4a6ed24ec729ffdc2cR113 (because a user should not care about the impl details ;-)